pdns/dnspcap.hh

```diff
@@ -115,7 +115,9 @@ public:
 
   pdns_pcap_file_header d_pfh;
   unsigned int d_runts, d_oversized, d_correctpackets, d_nonetheripudp;
-  char d_buffer[32768];
+  alignas (struct ip) char d_readbuffer[32768];
```

Comment:
To compensate for the part being chopped by the `d_bufsize` calculation maybe this?
```suggestion
  alignas (struct ip) char d_readbuffer[32768 + alignof(struct ip)];
```
or is it already oversized?

Reply:
It is already way oversized, and I do reduce `d_bufsize` to avoid overruns, so I think we're good..